_config.yml

```diff
@@ -153,3 +154,19 @@ publication:
     video: Videos
     training: Training
     book: Books
+
+############################################################
+
+# https://github.com/jekyll/jekyll-archives/blob/master/docs/configuration.md
+jekyll-archives:
+  enabled: 
+    - tags
+  layouts: 
+    tag: tag-archive
+  permalinks:
+    year: '/:year/'
+    month: '/:year/:month/'
+    day: '/:year/:month/:day/'
+    tag: '/tag/:name/'
```

Comment:
I'd do `/blog/tag/:name/` or are we planning to tag non blog pages and have them visible across?

Reply:
the jekyll-archives plugin just do it for posts so maybe best to just put under blog/tags/